frontend/convex/utils/schemaMapper.ts

```
@@ -0,0 +1,108 @@
+        probability: Number(record.probability || 0),
+        impactedParties: Array.isArray(record.impactedParties)
+          ? record.impactedParties
+          : safeJsonParse(record.impactedParties || record.impacted_parties, []),
+        chartSummary: record.chartSummary
+          ? safeJsonParse(record.chartSummary || record.chart_summary, undefined)
+          : undefined,
+      };
+
+    case "current_election_results":
+      // Ensure lastPull is a valid ISO-8601 string or use current timestamp
+      const lastPull = record.lastPull || record.last_pull;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)